src/components/layouts/EmptyState/DashboardEmptyState.tsx

```diff
@@ -0,0 +1,50 @@
+import { PrimaryBtn } from '@/components/buttons/PrimaryBtn'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { AddIcon, TasksListIcon } from '@/icons'
+import { setShowModal } from '@/redux/features/createTaskSlice'
+import store from '@/redux/store'
+import { UserType } from '@/types/interfaces'
+import { SxCenter } from '@/utils/mui'
+import { Box, Stack, Typography } from '@mui/material'
+
+const DashboardEmptyState = ({ userType }: { userType: UserType }) => {
+  return (
+    <AppMargin size={SizeofAppMargin.LARGE} py="20px">
+      <Box
+        sx={{
+          display: 'flex',
+          height: '80vh',
+          ...SxCenter,
+        }}
+      >
+        <Stack rowGap={'20px'} direction={'column'} sx={{ width: '453px' }}>
+          <Stack rowGap={'12px'} direction={'column'}>
+            <TasksListIcon />
+
+            <Typography variant="lg" lineHeight={'32px'}>
+              {userType == UserType.INTERNAL_USER ? " You don't have any tasks yet" : 'No tasks assigned'}
+            </Typography>
+            <Typography variant="bodyLg" sx={{ color: (theme) => theme.color.gray[500] }}>
+              {userType == UserType.INTERNAL_USER
+                ? 'Tasks will be shown here after they’re created. You can create a new task below.'
+                : 'Tasks will show here once they’ve been assigned to you. '}
```

Comment:
In the client side as well, the font size and icon background have inconsistency. Please verify properly with the Figma design .

In this PR - 

![Screenshot from 2024-07-11 09-36-53](https://github.com/copilot-platforms/tasks-app/assets/38468429/b716e331-7a94-4c48-85f5-499348d5ec99)

In Figma - 

![Screenshot from 2024-07-11 09-38-10](https://github.com/copilot-platforms/tasks-app/assets/38468429/fd5a2be5-7d62-4dbe-9da5-683cb07b3335)